go-controller/pkg/ovn/controller/egressservice/egressservice_zone_service.go

```diff
@@ -192,20 +191,14 @@ func (c *Controller) allEndpointsFor(svc *corev1.Service) (
 	return
 }
 
-func createIPAddressNetSlice(v4ips, v6ips []string) []net.IP {
-	ipAddrs := make([]net.IP, 0)
-	for _, ip := range v4ips {
-		ipNet := net.ParseIP(ip)
-		ipAddrs = append(ipAddrs, ipNet)
-	}
-	for _, ip := range v6ips {
-		ipNet := net.ParseIP(ip)
-		ipAddrs = append(ipAddrs, ipNet)
-	}
+func createIPAddressSetSlice(v4ips, v6ips []string) []string {
```

Comment:
nit: this is not a set right?

And you could just `return append(v4ips, v6ips...)`